common/option-model.cpp

```diff
@@ -239,7 +240,7 @@ bool option_model::draw_combobox( notifications_model & model,
 
     ImGui::SameLine();
     if( new_line )
```

Comment:
It is a parameter, is it always true?

Reply:
Yes, it receives true in the parameters of a function.
We call the function only once without sending it a boolean parameter.

Reply:
If it's always try on DQT + RS-Viewer then we can remove it sure

Reply:
But on another side in the future, if somebody adds a new call with `draw(..., ..., false)` it will create unexpected behavior. Sorry, let's leave it as is it now.